projects/noloan/firebase/src/main/java/hasadna/noloan/firebase/DbMessages.java

```diff
@@ -123,5 +112,26 @@ public void notifyListeners(int index, SmsMessage smsMessage, Type type) {
     // SmsMessage parameter is passed for cases the index already had been removed from the db list
     void messageRemoved(int index, SmsMessage smsMessage);
   }
+
+  // static function to search a given list
+  public static int findSmsByBody(SmsMessage sms, ArrayList<SmsMessage> list) {
```

Comment:
I think `findSms()` as you proposed before is better :+1: 
Perhaps we can agree as a convention that `findSms()` is by body & sender?

This function finds an SMS by its body AND sender, so `findSmsByBody()` is a little confusing IMO...

Reply:
What do you suggest the for a better name?